qbft/state.go

```diff
@@ -28,22 +24,22 @@ type IConfig interface {
 	GetNetwork() Network
 	// GetTimer returns round timer
 	GetTimer() Timer
+	// GetSignatureVerifier returns the signature verifier for operator signatures
+	GetSignatureVerifier() types.SignatureVerifier
```

Comment:
if this will be part of spec it will limit the implementation from creating its own version for example optimization. (we're using openssl rsa for example) also since because of this now rsa sigs are verified twice (once in msg validation and once in consensus) we plan to create a single verifier that is shared and cache inside to reduce it in the future.

Reply:
@y0sher 
you can use a different lib as long as it has the same behavior (pass json tests)

Reply:
I just want to have it written that signature verification is configurable as everything else in the struct.
Bad configuration can also happen on the timer, wrecking liveness.

Besides those are all protocol configurations that need consensus (and be defined by us) and not something nodes can just change as they wish